contracts/src/Timelock.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/governance/TimelockController.sol";
+
+/// @title TimelockController
+/// @notice A timelock controller for the StakeTable contract
+contract Timelock is TimelockController {
+    constructor(
+        uint256 minDelay,
+        address[] memory proposers,
+        address[] memory executors,
+        address admin
+    ) TimelockController(minDelay, proposers, executors, admin) { }
```

Comment:
if it's the same, we don't need to re-declare it, right?

Reply:
i know we need this contract to inherit, but we don't need to declare this constructor? if it's the same as the parent contract. 

i guess it doesn't hurt. so nvm my comment.

Reply:
I don't quite understand this. If I change

```diff
diff --git a/justfile b/justfile
index feecdd883..298ebbe92 100644
--- a/justfile
+++ b/justfile
@@ -143,7 +143,7 @@ build-docker-images:
     scripts/build-docker-images-native

 # generate rust bindings for contracts
-REGEXP := "^LightClient(V\\d+)?$|^LightClientArbitrum(V\\d+)?$|^FeeContract$|PlonkVerifier(V\\d+)?$|^ERC1967Proxy$|^LightClient(V\\d+)?Mock$|^StakeTable$|^EspToken$|^Timelock$"
+REGEXP := "^LightClient(V\\d+)?$|^LightClientArbitrum(V\\d+)?$|^FeeContract$|PlonkVerifier(V\\d+)?$|^ERC1967Proxy$|^LightClient(V\\d+)?Mock$|^StakeTable$|^EspToken$|^TimelockController$"
 gen-bindings:
     # Update the git submodules
     git submodule update --init --recursive
```

I do get bindings for the TimelockController.

Reply:
We also do this for `ERC1967Proxy`.